### ISSUE TITLE
support sass-loader version 8

### ISSUE THREAD

**Issue by mehdiraized** [prncc/antd-scss-theme-plugin#64]
plz update sass loader

**Comment by nodeworks** [comment]
I created a fork of this and provided a package on npm of this fork to support sass-loader v8:

[@nodeworks/antd-scss-theme-plugin](https://github.com/nodeworks/antd-scss-theme-plugin)

**Comment by mehdiraized** [comment]
Tnx man.
plz change 
```
import AntdScssThemePlugin from 'antd-scss-theme-plugin';
```
to 
```
import AntdScssThemePlugin from '@nodeworks/antd-scss-theme-plugin';
```
in your repository

**Comment by nirvdrum** [comment]
I find trying to pick dependency versions that satisfies everyone's dependency graph is hard. But, I'll note that CRA 3.3.0 now requires sass-loader 8.

**Comment by sangaline** [comment]
@nodeworks Would you be interested in becoming a maintainer for this project instead of maintaining a fork?

**Comment by nodeworks** [comment]
@sangaline yes i can definitely help out there if needed. let me know where i can help!

**Comment by mehdiraized** [comment]
> I created a fork of this and provided a package on npm of this fork to support sass-loader v8:
> 
> [@nodeworks/antd-scss-theme-plugin](https://github.com/nodeworks/antd-scss-theme-plugin)

plz update your package for support less-loader v6

**Comment by denistex** [comment]
@mehdiraized check out this fork:

https://github.com/atb-tech/antd-scss-theme-plugin

It supports both sass-loader v8 and less-loader v6.

**Comment by mehdiraized** [comment]
> @mehdiraized check out this fork:
> 
> https://github.com/atb-tech/antd-scss-theme-plugin
> 
> It supports both sass-loader v8 and less-loader v6.

not supported
